web/src/OverviewTable.tsx

```diff
@@ -134,37 +139,38 @@ const OverviewTableMenu = styled.section`
 `
 
 const ResourceTable = styled.table`
-  margin-top: ${SizeUnit(0.5)};
-  border-collapse: collapse;
-  border: 1px ${Color.gray40} solid;
```

Comment:
since the table border moved to the accordion wrapper, tables without label groups don't have a border.

![Screen Shot 2022-03-08 at 11 04 05 AM](https://user-images.githubusercontent.com/23283986/157279741-de0b4045-ff02-4259-b68d-a00d278c358b.png)

Reply:
fixed!